expert/ezproxy.cfg

```diff
@@ -450,13 +450,20 @@ DJ erudit.org
 # Fold3
 # Not using https://help.oclc.org/Library_Management/EZproxy/Database_stanzas/Fold3
 # Because of issue https://phabricator.wikimedia.org/T256107
+AnonymousURL -OPTIONS +https://api.fold3.com:9080/*
+AnonymousURL +https://img.fold3.com/*
```

Comment:
I just have one small question: why are these lines before the title? Do they have to be before it?

Reply:
AnonymousURL is a position-dependent directive, meaning that it applies to any configuration that follows, rather than being scoped to an individual database stanza. Stanza directives go inside the stanza and position-dependent directives go outside of stanzas.

Reply:
Got it!